marketpulse/main/models.py

```diff
@@ -66,7 +67,12 @@ class Carrier(models.Model):
     country = CountryField(null=True, blank=True)
 
     def __unicode__(self):
-        return self.name
+        all_countries = dict(countries)
+        country = self.country
+        if self.country.code in all_countries:
+            country = all_countries[self.country.code]
+
+        return '{0}, {1}, {2}'.format(self.name, country, self.parent_operator)
```

Comment:
Do we want to show ``self.parent_operator``? It might lead to lengthy strings and it will not show up correctly in the menu.